### PR TITLE
Added daily profit telegram command

### PR DESCRIPTION
**Pull request by stephendade**
Add a Telegram command that shows the user their per-day profit/loss (in BTC)

### CODE REVIEW

**Comment by glonlas** [comment]
Nice feature,  I like it.

Few comments and suggestions:
1. it will be great to add the `/daily` command into the telegram `/help`
2. the number of day will be considered as a string and must be cast to Int. Then we can check the value is bigger than 0.
3. `'<b>Daily Profit:</b>\n{}'`, adding the parameter `parse_mode=ParseMode.HTML` to send_msg. Otherwise, Telegram will display the tag `<b></b>`
4. we can display the result in a table format
5. add the helper in the Telegram keyboard

## Message to add in _help() (suggestion 1)
Update `_help()` method.
```python
*/daily [n]:* `Shows profit or loss per day, over the last n (default 7) days`
```

## _daily() method suggestion to fix str to int  (suggestions 2, 3, 4)
```python
@authorized_only
def _daily(bot: Bot, update: Update) -> None:
    """
    Handler for /daily <n>
    Returns a daily profit (in BTC) over the last n days.
    Default is 7 days
    :param bot: telegram bot
    :param update: message update
    :return: None
    """
    Trade.query.order_by(Trade.close_date).all()
    today = date.today().toordinal()
    profit_days = {}
    
    try:
        timescale = int(update.message.text.replace('/daily', '').strip())
    except:
        timescale = 7

    if isinstance(timescale, int) and timescale > 0:
        for day in range(0, timescale):
            #need to query between day+1 and day-1
            nextdate = date.fromordinal(today-day+1)
            prevdate = date.fromordinal(today-day-1)
            trades = Trade.query.filter(between(Trade.close_date, prevdate, nextdate)).all()
            curdayprofit = 0
            for trade in trades:
                curdayprofit += trade.close_profit * trade.stake_amount
            profit_days[date.fromordinal(today-day)] = curdayprofit

        # Display in table format
        stats = []
        for key, value in profit_days.items():
            stats.append([key, str(value) + ' BTC'])
        stats = tabulate(stats, headers=['Day', 'Profit'], tablefmt='simple')

        message = '<b>Daily Profit over the last {} days</b>:\n<pre>{}</pre>'.format(timescale, stats)
        send_msg(message, bot=bot, parse_mode=ParseMode.HTML)
        return
    else:
        send_msg('*Daily [n]:* `must be an integer greater than 0`', bot=bot)
        return
```

## Add the /daily helper in telegram keyboard in send_msg() (suggestion 5)
Update `send_msg()`method.
```python
    keyboard = [['/daily', '/profit', '/performance','/balance'],
                ['/count', '/status table', '/status'],
                ['/start', '/stop', '/help']]
```

**Comment by shusso** [comment]
@stephendade Thanks for the PR, nice stuff 👍

File: freqtrade/rpc/telegram.py [hunk 1]
```diff
@@ -206,7 +207,44 @@ def _status_table(bot: Bot, update: Update) -> None:
...
+    :param bot: telegram bot
+    :param update: message update
+    :return: None
+    """
+    trades = Trade.query.order_by(Trade.close_date).all()
+    today = date.today().toordinal()
+    profit_days = {}
+    
+    try:
+        timescale = update.message.text.replace('/daily', '').strip()
```
Comment (shusso) [inline freqtrade/rpc/telegram.py]: You can cast this to `int` like @glonlas suggested

File: freqtrade/rpc/telegram.py [hunk 2]
```diff
@@ -206,7 +207,44 @@ def _status_table(bot: Bot, update: Update) -> None:
...
+    except:
+        timescale = 7
+
```
Comment (shusso) [inline freqtrade/rpc/telegram.py]: If you want to support positive integers I'd suggest to make the check at this point and return like @glonlas suggested. So e.g.
```
if timescale < 0:
  send_msg('*Daily [n]:* `must be an integer greater than 0`', bot=bot)
  return
```

**Comment by glonlas** [comment]
The result is not great neither readible. We should change it

**Comment by stephendade** [comment]
Thanks for the comments @glonlas and @shusso! I've implemented the changes.

Ended up making the /daily command to require a number, rather than default to 7 - I think it mazkes thing more obvious to the user that way,

**Comment by shusso** [comment]
Solid stuff @stephendade 👍 Merging

**Comment by devalias** [comment]
just added this feature, nice.  when I click the /daily button it gives that message: "Daily <n>: must be an integer greater than 0".  I think clicking the button should at least give back daily for 1 day.

**Pull request merged**